devtools/server/actors/replay/module.js

```diff
@@ -273,6 +277,17 @@ function setSourceMap({
   });
 }
 
+function checkRecordingCreated() {
+  if (RecordReplayControl.isRecordingCreated()) {
+    clearTimeout(timer);
+    Services.cpmm.sendAsyncMessage("RecordReplayRecordingCreated", {
+      recordingId: RecordReplayControl.recordingId(),
+    });
+  }
+}
+const timer = setTimeout(checkRecordingCreated, 250);
+checkRecordingCreated();
```

Comment:
This logic seems strange, we only check immediately and then again after 250 ms?

Reply:
Oh woops that's what I get for throwing this together to get feedback without testing it.

This was meant to be `setInterval` not `setTimeout`, so we'd be polling every 250ms until it is created.